packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.tsx

```diff
@@ -113,12 +123,32 @@ export class GuxPopover {
     ) {
       this.dismiss();
     }
+
+    if (clickedForElement) {
+      this.popupElement?.showPopover();
```

Comment:
I was getting an error in the popover test file that `this.popupElement?.showPopover` doesn't not exist and I tried to think of a way to mock up this method with jest but was having some trouble. Do you all think using a `?.` here would be ok? Only worry I have is that this would swallow any errors of this method not invoking when it always should in the front end. I think ideally we'd mock up this method because that seems cleaner to me, but wanted to see what you all thought.